mongodb/Package.md

```diff
@@ -1,155 +1,22 @@
-# Ballerina MongoDB Connector
-
 Connects to MongoDB from ballerina 
 
-## Module Overview
-
-The Mongo DB connector allows you to connect to a Mongo DB from Ballerina and perform various operations such as `getDatabaseNames`, `getCollectionNames`, `count`, `listIndices`, `find`, `insert`, `update`, and `delete`.
-
-## Prerequisites
-
-* A mongodb with username and password
+## Package Overview
 
-* Java 11 Installed <br/> Java Development Kit (JDK) with version 11 is required.
+The `ballerinax/mongodb` is a [Ballerina](https://ballerina.io/) connector for MongoDB.
 
-* Ballerina SLAlpha5 Installed <br/> Ballerina Swan Lake Alpha 5 is required.
+The Mongo DB connector allows you to connect to a Mongo DB from Ballerina and perform various operations such as `getDatabaseNames`, `getCollectionNames`, `count`, `listIndices`, `find`, `insert`, `update`, and `delete`.
```

Comment:
Let's use the above given description if it makes sense.

Reply:
Fixed by 89976637b8fed8c9a16842370da109f71b035569.